### PR TITLE
Msi ms7d25/rel v1.1.3

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Review by mkopec — APPROVED** [review]

File: docs/variants/msi_z690/releases.md [hunk 1]
```diff
@@ -14,6 +14,84 @@ For details about our release process please read
...
+- Incorrect C-states reported in ACPI unsupported by HW
+- Incorrect first timestamp format in ACPI FPDT
+
+### Known issues
+
+- [Windows 11 installer unable to detect i225 Ethernet NIC](https://github.com/Dasharo/dasharo-issues/issues/482)
+- [Cannot wake from suspend via RTC on QubesOS](https://github.com/Dasharo/dasharo-issues/issues/484)
+- [Windows 11 VBS (Virtualization-based Security) appears Not enabled on
+  System Information](https://github.com/Dasharo/dasharo-issues/issues/539)
+
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/releases.md:74]: please add: 
- https://github.com/Dasharo/dasharo-issues/issues/484
- https://github.com/Dasharo/dasharo-issues/issues/521
- https://github.com/Dasharo/dasharo-issues/issues/483

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z690/releases.md (hunk 1 above)
Comment (macpijan) [reply docs/variants/msi_z690/releases.md:74]: https://github.com/Dasharo/dasharo-issues/issues/483#issuecomment-1895454842

**Comment by macpijan** [comment]
@BeataZdunczyk 

> also add a link to the new test results spreadsheet that we have since this release: https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit?usp=sharing


Let's use a link that directly leads to results for given platform, from page is not that interesting for subscribers.

File: docs/variants/msi_z690/releases.md (hunk 1 above)
Comment (miczyg1) [reply docs/variants/msi_z690/releases.md:74]: Take a closer look at `Cannot wake from suspend via RTC on QubesOS` the issue 484 is already there.

Comment (miczyg1) [reply docs/variants/msi_z690/releases.md:77]: https://github.com/Dasharo/docs/pull/686/commits/f91807ba59f8f0d7c2b08ec1faba9a3311d888dc#diff-02c1965ab265070941dfe35aabf982d0946a2ee1545fac0763eda95ccf36162fR76

**Comment by miczyg1** [comment]
> @BeataZdunczyk
> 
> > also add a link to the new test results spreadsheet that we have since this release: https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit?usp=sharing
> 
> Let's use a link that directly leads to results for given platform, front page is not that interesting for subscribers.

https://github.com/Dasharo/docs/pull/686/commits/f91807ba59f8f0d7c2b08ec1faba9a3311d888dc#diff-02c1965ab265070941dfe35aabf982d0946a2ee1545fac0763eda95ccf36162fR17

**Review by BeataZdunczyk — APPROVED** [review]

File: docs/variants/msi_z690/releases.md [hunk 2]
```diff
@@ -11,11 +11,115 @@ For details about our release process please read
 [newsletter]{.md-button .md-button--primary .center}
 </center>
 
-Test results for this platform can be found
-[here](https://docs.google.com/spreadsheets/d/16wokQYhtS7XA1DQC3Om7FY-IImG6SZisGK7NnzyRGVY/edit?usp=sharing).
+## v1.1.3 - 2024-01-17
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/releases.md:14]: same as https://github.com/Dasharo/docs/pull/687#discussion_r1461583060

File: docs/variants/msi_z690/releases.md [hunk 3]
```diff
@@ -11,11 +11,115 @@ For details about our release process please read
...
+- Incorrect C-states reported in ACPI unsupported by HW
+- Incorrect first timestamp format in ACPI FPDT
+
+### Known issues
+
+- [Windows 11 installer unable to detect i225 Ethernet NIC](https://github.com/Dasharo/dasharo-issues/issues/482)
+- [Cannot wake from suspend via RTC on QubesOS](https://github.com/Dasharo/dasharo-issues/issues/484)
+- [Windows 11 VBS (Virtualization-based Security) appears Not enabled on System Information](https://github.com/Dasharo/dasharo-issues/issues/539)
+- [No ability to change active PCR banks with TPM PPI in FW](https://github.com/Dasharo/dasharo-issues/issues/521)
+
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/releases.md:77]: @miczyg1 https://github.com/Dasharo/docs/pull/687#discussion_r1461582142

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z690/releases.md (hunk 2 above)
Comment (miczyg1) [reply docs/variants/msi_z690/releases.md]: Changed: 190f0b1bebbf01ecd7833fcd1d2d8b842d8a222f

File: docs/variants/msi_z690/releases.md (hunk 3 above)
Comment (miczyg1) [reply docs/variants/msi_z690/releases.md:77]: Added 190f0b1bebbf01ecd7833fcd1d2d8b842d8a222f

**Comment by BeataZdunczyk** [comment]
@miczyg1 you can merge this release notes after fixing link checker errors https://github.com/Dasharo/docs/actions/runs/7610834739/job/20725011619?pr=686

**Comment by macpijan** [comment]
Sop links are broken, the same is on master, so this MR is in no way worse. We need to find out internally what has happened and fix it in a followup.

**Pull request merged**